README.md

```diff
@@ -227,7 +227,7 @@ Username is shown only when it's not the same as `$LOGNAME`, when you're connect
 
 | Variable | Default | Meaning |
 | :------- | :-----: | ------- |
-| `SPACESHIP_USER_SHOW` | `true` | Show user section |
+| `SPACESHIP_USER_SHOW` | `always` | Show user section. Options are `always`, `ssh` and `never`. |
```

Comment:
Defaulting to `always` is against the prompt behavior, I think `ssh` would be better as default.